src/main/java/org/influxdb/impl/InfluxDBImpl.java

```diff
@@ -53,7 +53,7 @@
  */
 public class InfluxDBImpl implements InfluxDB {
 
-  static final okhttp3.MediaType MEDIA_TYPE_STRING = MediaType.parse("text/plain");
+  static final MediaType MEDIA_TYPE_STRING = MediaType.parse("text/plain");
```

Comment:
IMO, the previous code was a bit more clear by stating that `MediaType` comes from `okhttp3` and not from another package (e.g. `javax.ws.rs.core.MediaType`).
I don't like also the name as there is no media type string (accordingly with the RFC) but this is another issue.